Samples/PowerShell/ShardManagement/CreatePerformanceCounterCategory.ps1

```diff
@@ -0,0 +1,43 @@
+﻿<#
+/********************************************************
+*                                                        *
+*   © Microsoft. All rights reserved.                    *
+*                                                        *
+*********************************************************/
+
+.SYNOPSIS
+    This script creates performance counter catagory and adds
+    specified user to 'Performance Monitor Users' local group.
+
+.NOTES
+    Author: Microsoft SQL Elastic Scale team
+    Last Updated: 5/6/2016
+
+.EXAMPLES
+    .\CreatePerformanceCounterCategory.ps1 `
+        -UserName 'domain\username'
+#>
+
+param(
+	[Parameter(Mandatory = $true)]
+	[string]
+	$UserName=""
+)
+$scriptPath = split-path -parent $MyInvocation.MyCommand.Path
+$dllNameWithPath = $scriptPath + "\Microsoft.Azure.SqlDatabase.ElasticScale.Client.dll";
```

Comment:
Replace line 27 with the below line to import the dll. There is code to locate the dll and download if necessary.

Import-Module $ScriptDir\ShardManagement -Force